contrib/opencensus-ext-azure/README.rst

```diff
@@ -114,11 +114,70 @@ You can enrich the logs with trace IDs and span IDs by using the `logging integr
         logger.warning('In the span')
     logger.warning('After the span')
 
+Metric
+~~~~~~
+
+The **OpenCensus Azure Monitor Metrics Exporter** allows you to export metrics to `Azure Monitor`_.
+
+Metrics Exporter Usage
+~~~~~~~~~~~~~~~~~~~~~~
+
+Metrics Exporter Prerequisites
+******************************
+
+* Create an Azure Monitor resource and get the instrumentation key, more information can be found `here <https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource>`_.
+* Put the instrumentation key in ``APPINSIGHTS_INSTRUMENTATIONKEY`` environment variable.
+
+Using the Metrics exporter
+*****************************
+
+    .. code:: python
+
+        import time
+
+        from opencensus.ext.azure import metric_exporter
+        from opencensus.stats import aggregation as aggregation_module
+        from opencensus.stats import measure as measure_module
+        from opencensus.stats import stats as stats_module
+        from opencensus.stats import view as view_module
+        from opencensus.tags import tag_map as tag_map_module
+
+
+        # The stats recorder
+        stats = stats_module.stats
+        view_manager = stats.view_manager
+        stats_recorder = stats.stats_recorder
+
+        PROBLEMS_SOLVED_MEASURE = measure_module.MeasureInt("problems_solved", "number of problems solved", "problems")
+        PROBLEMS_SOLVED_VIEW = view_module.View('problems_solved_view', "number of problems solved", [], PROBLEMS_SOLVED_MEASURE, aggregation_module.CountAggregation())
+
+        def main():
+            # Enable metrics
+            # Set the interval in seconds in which you want to send metrics
+            exporter = metric_exporter.new_metrics_exporter(export_interval=5)
+            view_manager.register_exporter(exporter)
+
+            view_manager.register_view(PROBLEMS_SOLVED_VIEW)
+            mmap = stats_recorder.new_measurement_map()
+            tmap = tag_map_module.TagMap()
+
+            mmap.measure_int_put(PROBLEMS_SOLVED_MEASURE, 1000)
+            mmap.record(tmap)
+            time.sleep(5)
+
+            print("Done recording metrics")
+
+
+        if __name__ == "__main__":
+            main()
+        
+
 References
 ----------
 
 * `Azure Monitor <https://docs.microsoft.com/azure/azure-monitor/>`_
 * `Examples <https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-azure/examples>`_
+* `Implementation <https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-azure/opencensus/ext/azure/metric_exporter>`_
```

Comment:
This is a bit confusing:
1. Why do we need to put a link to the implementation?
2. This is a doc for azure exporters, which is dedicated for metrics, the link is misleading.
3. We need to have a consistent use of plural form (e.g. `metric_exporter` and `new_metrics_exporter`.